lib/bundles.cf

```diff
@@ -301,6 +301,39 @@ bundle agent prunedir(dir, max_days)
       depth_search  => recurse("1");
 }
 
+bundle agent prunetree(dir, depth, max_days)
+# @brief Delete files and directories inside "dir" up to "depth" older than "max_days".
+# @depends delete tidy
+# @depends depth_search recurse_with_base
+# @depends file_select days_old
+# @param dir directory to examine for files
+# @param depth How many levels to descend
+# @param max_days maximum number of days old a files mtime is allowed to before deletion
+#
+# **Example:**
+#
+# ```cf3
+# bundle agent example
+# {
+#   vars:
+#     "dirs" slist => { "/tmp/logs", "/tmp/logs2" };
+#
+#   methods:
+#     "prunetree" usebundle => prunetree( @(dirs), inf, "1" );
+# }
+# ```
+{
+  files:
+      "$(dir)"
+      comment => "Delete files and directories under $(dir) up to $(depth)
+                    depth older than $(max_days)",
+
+      delete        => tidy,
+      file_select   => days_old( $(max_days) ),
```

Comment:
```
lib/files.cf-# @param days Number of days
lib/files.cf-{
lib/files.cf-      mtime       => irange(0,ago(0,0,"$(days)",0,0,0));
lib/files.cf-      file_result => "mtime";
lib/files.cf-}
```

Why do I read this as selecting whatever was modified up to these many days old? So the opposite of what we want.